docs/src/tutorials/low_level_2.md

```diff
@@ -0,0 +1,136 @@
+# Using `ahmc_bayesian_pinn_pde` with the `BayesianPINN` Discretizer for the Kuramoto–Sivashinsky equation
```

Comment:
Is this `ahmc_bayesian_pinn_pde` or `advancedhmc_pinn_pde`? Because I see `advancedhmc_pinn_pde` in the manuals page.

Reply:
```suggestion
# Using Bayesian PINNs for Uncertainty-Aware  solutions on the Kuramoto–Sivashinsky equation
```

Reply:
Why would this be low level? This is just a bpinn tutorial